sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -167,10 +167,36 @@ bool PeriodicExportingMetricReader::OnForceFlush(std::chrono::microseconds timeo
   }
 
   is_force_flush_notified_.store(false, std::memory_order_release);
+
+  if (timeout == std::chrono::steady_clock::duration::zero())
+  {
+  }
+  if (timeout_steady <= std::chrono::steady_clock::duration::zero())
+  {
+    // forceflush timeout, exporter force-flush won't be triggered.
+    result = false;
```

Comment:
Do we need reset result to false here?I think result will keep false and break the loop above when `timeout_steady <= std::chrono::steady_clock::duration::zero()`.
And if result is true and `timeout_steady <= std::chrono::steady_clock::duration::zero()`, result will also be set to false below.